### PR TITLE
Expose static ips and documentation to tenants

### PR DESCRIPTION
**Pull request by paroxp**

### CODE REVIEW

File: src/components/support/views.tsx [hunk 1]
```diff
@@ -1802,3 +1802,93 @@ export function SignUpPage(props: ISignupFormProperties): ReactElement {
...
+      </p>
+
+      <p className="govuk-body">
+        Download the:
+      </p>
+
+      <ul className="govuk-list govuk-list--bullet">
+        <li>
+          <a href="/downloads/non-crown-mou.pdf" className="govuk-link" download={true}>
+            GOV.UK PaaS memorandum of understanding for non-Crown bodies
```
Comment (kr8n3r) [inline src/components/support/views.tsx:1884]:
```suggestion
            GOV.UK PaaS memorandum of understanding for non-Crown bodies [PDF]
```

File: src/components/support/views.tsx [hunk 2]
```diff
@@ -1802,3 +1802,93 @@ export function SignUpPage(props: ISignupFormProperties): ReactElement {
...
+      </p>
+
+      <p className="govuk-body">
+        Download the:
+      </p>
+
+      <ul className="govuk-list govuk-list--bullet">
+        <li>
+          <a href="/downloads/crown-mou.pdf" className="govuk-link" download={true}>
+            GOV.UK PaaS memorandum of understanding for Crown bodies
```
Comment (kr8n3r) [inline src/components/support/views.tsx:1856]:
```suggestion
            GOV.UK PaaS memorandum of understanding for Crown bodies [PDF]
```

**Review by kr8n3r — COMMENTED** [review]
apart from adding document type as we do elsewhere, it's good to go

**Comment by 46bit** [comment]
@paroxp I was going to review this but there's a massive number of test failures. Could you take a look?

**Review by kr8n3r — APPROVED** [review]

**Comment by kr8n3r** [comment]
tested this locally, works as expected. code looks good. will merge in Rafal's absence

**Pull request merged**